crates/bevy_asset/src/path.rs

```diff
@@ -80,8 +80,10 @@ impl<'a> Display for AssetPath<'a> {
 
 #[derive(Error, Debug, PartialEq, Eq)]
 pub enum ParseAssetPathError {
```

Comment:
Non-blocking: we should add top-level docs to this struct.

Reply:
Top-level docs coming right up :+1: